yt_dlp/extractor/tiktok.py

```diff
@@ -767,7 +790,8 @@ def _video_entries_api(self, webpage, user_id, username):
             'max_cursor': 0,
             'min_cursor': 0,
             'retry_type': 'no_retry',
-            'device_id': ''.join(random.choices(string.digits, k=19)),  # Some endpoints don't like randomized device_id, so it isn't directly set in _call_api.
+            'device_id': ''.join(random.choices(string.digits, k=19)),
+            # Some endpoints don't like randomized device_id, so it isn't directly set in _call_api.
```

Comment:
revert unnecessary/unrelated formatting change
```suggestion
            'device_id': ''.join(random.choices(string.digits, k=19)), # Some endpoints don't like randomized device_id, so it isn't directly set in _call_api.
```